templates/advantage/index.html

```diff
@@ -367,7 +376,7 @@ <h4>Ubuntu 16.04 and 18.04 LTS</h4>
           {% endfor %}
         </div>
       <div class="col-12 u-align--right">
-        <a href="/advantage/subscribe" class="p-button--positive u-align--right" >Add new subscription</a>
+        <a href="/advantage/subscribe{% if is_test_backend %}?test_backend=true{% endif %}" class="p-button--positive u-align--right" >Add new subscription</a>
```

Comment:
I'm trying to remember the reasons why we implemented this `test_backend=true` feature. Can anyone remember? @sowasred2012 ?

If we're at the point where we're adding significant modifications to templates just to support our testing, I think maybe we should have another look at whether this is a good solution.

Reply:
This is the PR that introduced it https://github.com/canonical-web-and-design/ubuntu.com/pull/8888
I don't remember how the whole conversation went but I think it was because using `.env` to set the API url to the test backend when developing is bad practice as we should be as close to the real environment. so we agreed a query param was a good way to do it

Reply:
@nottrobin @ClementChaumel the reason was as soon as we were no longer working on a feature branch and were instead creating pull requests to be merged into master, the way that we had been setting the staging environment - hardcoding the appropriate values, then updating them to production when the feature branch was eventually merged into master - became very unwieldy. This solution seemed like a good compromise at the time, as Clem says.